src/context/wishlist.context.jsx

```diff
@@ -0,0 +1,43 @@
+import { createContext, useContext, useReducer } from "react";
+
+const ContextOfWishlist = createContext(null);
+
+const wishlistReducerFn = (state, action) => {
+  switch (action.payload) {
+    case "add_to_wishlist":
```

Comment:
this actions type can be placed inside a constants file and can be used any place we want